.github/workflows/run-qa-engine.yml

```diff
@@ -0,0 +1,30 @@
+name: Run QA Engine
+
+on:
+  workflow_dispatch:
+  schedule:
+    # 1pm UTC is 6am PDT.
+    # same time as Generate Build Report
+    - cron: "0 13 * * *"
+
+jobs:
+  run-qa-engine:
+    name: "Run QA Engine"
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+      - name: Setup Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+        with:
+          service_account_key: ${{ secrets.PROD_SPEC_CACHE_SA_KEY }}
+          export_default_credentials: true
```

Comment:
Is this a new secret?

Reply:
Nope, it's the one used by the deploy-oss-catalog.yml workflow, we use it to write to GCS.